static/src/javascripts/projects/common/modules/video/events.js

```diff
@@ -119,25 +121,6 @@ define([
         return action;
     }
 
-    function buildGoogleAnalyticsEvent(mediaEvent, metrics, canonicalUrl) {
```

Comment:
@akash1810 I didn't move this as it isn't shared between the events.js implementation and the new you-tube-tracking.js implemenation.